apps/documentation/.eslintrc.json

```diff
@@ -1,3 +0,0 @@
-{
-  "extends": ["next/core-web-vitals", "next/typescript"]
```

Comment:
removing this file since `eslint` configuration is done already at the root of the monorepo
also going to just use our normal linting rules instead of the `next` ones